assets/README.md

```diff
@@ -6,7 +6,7 @@ Each file represents an equipment type and is listed with the following attribut
 
 * _Make_ - Equipment manufacturer or commercial brand name.
 * _Model_ - Equipment model name or general identification label.
-* _Serial_ - Equipment serial number, needs to be unique within its set of equipment make & model.
+* _Serial_ - Equipment serial number, needs to be unique within its set of equipment make &amp; model.
```

Comment:
minor comment: can we just use `and` instead of `&`, just to make the readme "text-friendly"?

Reply:
Agreed `and` is tidier.